examples/default/variables.tf

```diff
@@ -1,29 +1,59 @@
 variable "ibmcloud_api_key" {
   type        = string
-  description = "The IBM Cloud API Key"
+  description = "The IBM Cloud API token this account authenticates to"
   sensitive   = true
 }
 
 variable "region" {
   type        = string
-  description = "Region to provision all resources created by this example"
+  description = "Region of the secrets manager instance"
   default     = "us-south"
 }
 
-variable "prefix" {
+variable "root_ca_name" {
   type        = string
-  description = "Prefix to append to all resources created by this example"
-  default     = "terraform"
+  description = "Name of the Root CA to create for a private_cert secret engine"
+  default     = "root-ca"
 }
 
-variable "resource_group" {
+variable "intermediate_ca_name" {
   type        = string
-  description = "An existing resource group name to use for this example, if unset a new resource group will be created"
-  default     = null
+  description = "Name of the Intermediate CA to create for a private_cert secret engine"
+  default     = "intermediate-ca"
+}
+
+variable "certificate_template_name" {
+  type        = string
+  description = "Name of the Certificate Template to create for a private_cert secret engine"
+  default     = "my-template"
```

Comment:
Can we rename it to something else? like `cert-template`?

Reply:
It's coming directly from internal version - this can be changed, but not in the initial PR that maintains parity with internal code base from a functional perspective.